src/main/msp/msp.c

```diff
@@ -2046,7 +2046,7 @@ case MSP_NAME:
         sbufWriteU8(dst, currentPidProfile->tpa_mode);
         sbufWriteU8(dst, currentPidProfile->tpa_rate);
         sbufWriteU16(dst, currentPidProfile->tpa_breakpoint);   // was currentControlRateProfile->tpa_breakpoint
-        // Added with API 1.46: dono if this is legit
+        // Added with API 1.47: dono if this is legit, TBC
```

Comment:
1.47 is release 4.6. If this is intended I can change the milestone.

Reply:
Thx for pointing that out. I just corrected it.